parser-typechecker/src/Unison/Codebase/Causal.hs

```diff
@@ -339,25 +339,39 @@ unsafeMapHashPreserving f c = case c of
   Cons h e (ht, tl) -> Cons h (f e) (ht, unsafeMapHashPreserving f <$> tl)
   Merge h e tls -> Merge h (f e) $ Map.map (fmap $ unsafeMapHashPreserving f) tls
 
+data FoldHistoryResult a = Satisfied a | Unsatisfied a deriving (Eq,Ord,Show)
+
 -- foldHistoryUntil some condition on the accumulator is met,
 -- attempting to work backwards fairly through merge nodes
 -- (rather than following one back all the way to its root before working
 -- through others).  Returns Unsatisfied if the condition was never satisfied,
 -- otherwise Satisfied.
-data FoldHistoryResult a = Satisfied a | Unsatisfied a deriving (Eq,Ord,Show)
```

Comment:
moved above the comment

Reply:
The comment is on the function

Reply:
👍

Reply:
Yeah, not a critique, just when viewing the diff it looked like the moved version appeared from nowhere, because this had originally been buried in with the function.